hooks/useActivity.ts

```diff
@@ -1,15 +1,21 @@
 import { useMutation, useQuery, useQueryClient } from "@tanstack/react-query";
 import {
   copyActivitiesRequest,
-  createActivityRequestForCitizen,
+  createActivityCitizen,
+  createActivityGrade,
   deleteRequest,
   fetchActivityRequest,
-  fetchByDateRequestForCitizen,
+  fetchByDateCitizen,
+  fetchByDateGrade,
   toggleActivityStatusRequest,
   updateRequest,
 } from "../apis/activityAPI";
+import { useWeekplan } from "../providers/WeekplanProvider";
 
-import { useCitizen } from "../providers/CitizenProvider";
+/* It's highly recommended to read the docs at https://tanstack.com/query/latest
```

Comment:
What's this for?

Reply:
Caching library that we use, hard to document everything in the codebase. So we just linked the docs